lighty-core/lighty-controller/src/main/java/io/lighty/core/controller/api/AbstractLightyModule.java

```diff
@@ -190,6 +190,7 @@ public synchronized ListenableFuture<Boolean> shutdown() {
     @Override
     public final boolean shutdown(final long duration, final TimeUnit unit) {
         try {
+            LOG.info("Stopping lighty module: {}", getClass().getSimpleName());
```

Comment:
This is not appropriate here. We have log like this in `shutdown` method already.